### PR TITLE
1.12.2 notes amendment

### PR DESCRIPTION
**Pull request by howardjohn**
Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [ ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Review by craigbox — APPROVED** [review]

File: content/en/news/releases/1.12.x/announcing-1.12.2/index.md [hunk 1]
```diff
@@ -18,6 +18,9 @@ This release fixes the security vulnerability described in our January 18th post
 - __[CVE-2022-21679](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2CVE-2022-21679i])__:
   Istio 1.12.0/1.12.1 contain a vulnerability where 1.11 proxy configuration is generated incorrectly affecting the `hosts` and `notHosts` field in the authorization policy.
 
+- __[CVE-2022-21701](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2CVE-2022-21679i])__:
+  Istio version 1.12.0 and 1.12.1 are vulnerable to a privilege escalation attack. Users who have `CREATE` permission for `gateways.gateway.networking.k8s.io` objects can escalate this privilege to create other resources that they may not have access to, such as `Pod`.
```
Comment (craigbox) [inline content/en/news/releases/1.12.x/announcing-1.12.2/index.md:22]:
```suggestion
  Istio versions 1.12.0 and 1.12.1 contain a vulnerability where configuration for proxies at version 1.11 is generated incorrectly, affecting the `hosts` and `notHosts` field in the authorization policy.

- __[CVE-2022-21701](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2CVE-2022-21679i])__:
  Istio versions 1.12.0 and 1.12.1 are vulnerable to a privilege escalation attack. Users who have `CREATE` permission for `gateways.gateway.networking.k8s.io` objects can escalate this privilege to create other resources that they may not have access to, such as `Pod`.
```

File: content/en/news/releases/1.12.x/announcing-1.12.2/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ aliases:
     - /news/announcing-1.12.2
 ---
 
-This release fixes the security vulnerability described in our January 18th post, [ISTIO-SECURITY-2022-001](/news/security/istio-security-2022-001) as well as a few minor bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2.
+This release fixes the security vulnerability described in our January 18th post, [ISTIO-SECURITY-2022-001](/news/security/istio-security-2022-001), [ISTIO-SECURITY-2022-002](/news/security/istio-security-2022-002), and a few minor bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2.
```
Comment (craigbox) [inline content/en/news/releases/1.12.x/announcing-1.12.2/index.md:12]:
```suggestion
This release fixes security vulnerabilities described on January 18th post ([ISTIO-SECURITY-2022-001](/news/security/istio-security-2022-001) and [ISTIO-SECURITY-2022-002](/news/security/istio-security-2022-002)) and includes minor bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2.
```

File: content/en/news/releases/1.12.x/announcing-1.12.2/index.md [hunk 3]
```diff
@@ -9,7 +9,7 @@ aliases:
     - /news/announcing-1.12.2
 ---
 
-This release fixes the security vulnerability described in our January 18th post, [ISTIO-SECURITY-2022-001](/news/security/istio-security-2022-001), [ISTIO-SECURITY-2022-002](/news/security/istio-security-2022-002), and a few minor bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2.
+This release fixes security vulnerabilities described on January 18th post ([ISTIO-SECURITY-2022-001](/news/security/istio-security-2022-001) and [ISTIO-SECURITY-2022-002](/news/security/istio-security-2022-002)) and includes minor bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2.
```
Comment (craigbox) [inline content/en/news/releases/1.12.x/announcing-1.12.2/index.md:12]:
```suggestion
This release fixes security vulnerabilities described on January 18th ([ISTIO-SECURITY-2022-001](/news/security/istio-security-2022-001) and [ISTIO-SECURITY-2022-002](/news/security/istio-security-2022-002)) and includes minor bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2.
```

Comment (craigbox) [reply content/en/news/releases/1.12.x/announcing-1.12.2/index.md:12]: forgot one sorry

Comment (craigbox) [reply content/en/news/releases/1.12.x/announcing-1.12.2/index.md:12]: feel free to remove DNM when done

File: content/en/news/releases/1.12.x/announcing-1.12.2/index.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ aliases:
     - /news/announcing-1.12.2
 ---
 
-This release fixes security vulnerabilities described on January 18th ([ISTIO-SECURITY-2022-001](/news/security/istio-security-2022-001) and [ISTIO-SECURITY-2022-002](/news/security/istio-security-2022-002)) and includes minor bug fixes to improve robustness. This release note describes what’s different between Istio 1.12.1 and Istio 1.12.2.
```
Comment (craigbox) [inline content/en/news/releases/1.12.x/announcing-1.12.2/index.md]: this is a load bearing bracket, it should render as 


> This release fixes security vulnerabilities described on January 18th (ISTIO-SECURITY-2022-001 and ISTIO-SECURITY-2022-002) and includes minor bug fixes to improve robustness. 

Was it breaking the lint?

Comment (howardjohn) [reply content/en/news/releases/1.12.x/announcing-1.12.2/index.md]: No, I just screwed it up - should be good now. thanks for the pointer

**Pull request merged**